src/components/TodoHistoryListModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+
+import { useSelector } from "react-redux";
+import { AppState } from "src/store/rootReducer";
+
+import { colors, getShadow } from "src/styles";
+import { css, StyleSheet } from "aphrodite";
+import { i18n } from "src/locale";
+
+import { ITodo } from "src/types";
+
+import { StyledText } from "./StyledText";
+import { HistoryTodo } from "./Todo";
+
+export interface Props {}
+
+export const TodoHistoryListModal: FC<Props> = () => {
+  const { todos } = useSelector((state: AppState) => state.todo);
+  return (
+    <>
+      <div className={css(styles.title)}>
+        <StyledText style={styles.titleText}>
+          {i18n.t("todo:history")}
+        </StyledText>
+      </div>
+      <div className={css(styles.wrapper)}>
+        {!todos.length ? (
```

Comment:
it wouldnt work as u expect

Reply:
Yes, you have right. I chose wrong array, historyTodos is correctly in this case.